.gitignore

```diff
@@ -122,3 +122,5 @@ config\.json
 *.ods
 temp_config\.json
 log
+
+config/trading_state_history\.json
```

Comment:
This is not possible (this path) with the current docker volume management.

Reply:
A good feature to solve this problem can be a folder for config.json and this file.